wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/exceptions/GitConflictException.java

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git.exceptions;
+
+import org.eclipse.che.api.git.GitException;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Git Conflict Exception
+ *
+ * @author Yossi Balan ( yossi.balan@sap.com)
+ */
+public class GitConflictException extends GitException {
+
+    private List<String> conflictingPaths;
```

Comment:
@vinokurig - it is ok to have null because we have only setter from the constructor